backend/sirius-web-diagrams-layout/src/main/java/org/eclipse/sirius/web/diagrams/layout/ELKDiagramConverter.java

```diff
@@ -365,6 +368,7 @@ private void convertBorderNode(Node borderNode, ElkNode elkNode, Map<String, Elk
         if (borderNode.getStyle() instanceof ImageNodeStyle) {
             Size imageSize = this.imageNodeStyleSizeProvider.getSize((ImageNodeStyle) borderNode.getStyle());
             elkPort.setDimensions(imageSize.getWidth(), imageSize.getHeight());
+            ((ImageNodeStyle)borderNode.getStyle()).getImageNodeSides().forEach(imageNodeSide -> elkPort.setProperty(CoreOptions.PORT_SIDE, PortSide.valueOf(imageNodeSide.name())));
```

Comment:
I don't think that this is enough since we have two layout algorithms. This will only be called during the initial layout and when a full layout is triggered with the `Arrange All` button. This constraint would also need to be taken into account in the incremental refresh.

Reply:
What happens if you use a tool which creates a new border node? Is this constraint respected or do you have to perform an `Arrange All` for it to be taken into account?

Reply:
incremental refresh border node problem has been resolved by #832 .

Reply:
@sbegaudeau incremental refresh border node problem has been resolved by #832 .